core/server/themes/active.js

```diff
@@ -84,6 +92,30 @@ class ActiveTheme {
     config(key) {
         return this._config[key];
     }
+
+    mount(blogApp) {
```

Comment:
Hmm the active theme class is now responsible to modify the express app + requiring the express hbs. I would prefer leaving the hbs logic in the theme middleware to decouple it.